src/testing.ts

```diff
@@ -80,31 +80,33 @@ const expectedResults = new Map<string, ResultType>(
     ['example_contracts/function_with_nested_return', 'Success'],
     ['example_contracts/functionArgumentConversions', 'Success'],
     ['example_contracts/functionInputs/arrayTest/arrayArrayArray', 'Success'],
-    ['example_contracts/functionInputs/arrayTest/arrayArrayDynArray', 'NotSupportedYet'],
+    ['example_contracts/functionInputs/arrayTest/arrayArrayBytes', 'WillNotSupport'],
+    ['example_contracts/functionInputs/arrayTest/arrayArrayDynArray', 'WillNotSupport'],
     ['example_contracts/functionInputs/arrayTest/arrayArrayStruct', 'Success'],
-    ['example_contracts/functionInputs/arrayTest/arrayDynArrayArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/arrayTest/arrayDynArrayDynArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/arrayTest/arrayDynArrayStruct', 'NotSupportedYet'],
+    ['example_contracts/functionInputs/arrayTest/arrayDynArrayArray', 'WillNotSupport'],
+    ['example_contracts/functionInputs/arrayTest/arrayDynArrayDynArray', 'WillNotSupport'],
+    ['example_contracts/functionInputs/arrayTest/arrayDynArrayStruct', 'WillNotSupport'],
     ['example_contracts/functionInputs/arrayTest/arrayStructArray', 'Success'],
-    ['example_contracts/functionInputs/arrayTest/arrayStructDynArray', 'NotSupportedYet'],
+    ['example_contracts/functionInputs/arrayTest/arrayStructDynArray', 'WillNotSupport'],
     ['example_contracts/functionInputs/arrayTest/arrayStructStruct', 'Success'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayDynArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayStruct', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayDynArray', 'NotSupportedYet'],
-    ['example_contracts/functionInputs/dynArrayTest/dynArrayDynArrayStruct', 'NotSupportedYet'],
+    ['example_contracts/functionInputs/dynArrayTest/dynArrayArrayArray', 'Success'],
```

Comment:
Array Child of DynArray here. Should not be supported since no pointer to tuple type. Unless the contract is wrong? The input to the contract is `uint[3][3][] memory y`

Reply:
Looks like pointers to tuples are supported.